src/reuser/ui/agent-reuser.php

```diff
@@ -26,7 +26,11 @@
 class ReuserAgentPlugin extends AgentPlugin
 {
   const UPLOAD_TO_REUSE_SELECTOR_NAME = 'uploadToReuse';
-    
+  const REUSE_NONE = 0;
+  const REUSE_ENHANCED = 2;
+  const REUSE_MAIN = 4;
+  const REUSE_ALL = 8;
```

Comment:
if REUSE_ALL is intended to be `REUSE_ENHANCED|REUSE_MAIN` it should be 6; otherwise someone could assume that `REUSE_ALL|REUSE_ENHANCED|REUSE_MAIN` is possible